utils/python/CIME/XML/env_mach_pes.py

```diff
@@ -95,6 +95,8 @@ def get_total_tasks(self, comp_classes):
             pstrid = self.get_value("PSTRID_%s"%comp)
             tt = rootpe + (ntasks - 1) * pstrid + 1
             total_tasks = max(tt, total_tasks)
+        if total_tasks < self.get_value("PES_PER_NODE"): # all PEs fit in 1 node
```

Comment:
It's not being changed in the code.   The default pelayout defined in driver_cpl/cime_config/config_pes.xml
is to use one full node (value -1 where negative means node count rather than task count) .  Acme gets its pe layouts from a different file and has a value of 8.   How did you ask for 8 MPI ranks?